portfolio/src/main/webapp/index.html

```diff
@@ -2,11 +2,20 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Mihai Preda's portfolio</title>
     <link rel="stylesheet" href="style.css">
+
     <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/fe29c7813b.js" crossorigin="anonymous"></script>
```

Comment:
What's this for?

Reply:
That is for the icons in the header (mail, phone, facebook, linkedin and github)